tests/test-verify-pool-members.yml

```diff
@@ -70,6 +70,23 @@
     loop_var: pv
   when: storage_test_pool.type == 'lvm'
 
+- name: Check that blivet supports PV grow to fill
+  ansible.builtin.script: >-
+    scripts/does_library_support.py
+    blivet.formats.lvmpv.LVMPhysicalVolume.grow_to_fill
+  register: grow_supported
+  changed_when: false
```

Comment:
Using `script` with python scripts is tricky because the python interpreter path is different on every platform.  The `snapshot` role uses `script` and has quite a bit of logic to determine which python interpreter to use.

https://github.com/linux-system-roles/snapshot/blob/main/tasks/main.yml#L21
The default value for `__snapshot_python` is defined here: https://github.com/linux-system-roles/snapshot/blob/main/vars/main.yml#L6

override for el7: https://github.com/linux-system-roles/snapshot/blob/main/vars/RedHat_7.yml#L4 and https://github.com/linux-system-roles/snapshot/blob/main/vars/CentOS_7.yml#L4

override for el8: https://github.com/linux-system-roles/snapshot/blob/main/vars/CentOS_8.yml#L4 and https://github.com/linux-system-roles/snapshot/blob/main/vars/RedHat_8.yml#L4

You might ask - well, why doesn't `script` use the Ansible python interpreter on the remote system?  If you can figure out how to do that, good - I could not figure out how to do that, which is why we implemented the `snapshot` role the way we did.

Reply:
I did a bit of testing and it seems that the value we are looking for is in `ansible_python.executable`.
I have added
```
      args:
        executable: "{{ ansible_python.executable }}"
```
to see how it fares.